lotus-soup/compositions/composition-local-drand.toml

```diff
@@ -9,6 +9,12 @@
   builder = "docker:go"
   runner = "local:docker"
 
+[global.build]
+  selectors = ["testground"]
+
+[global.run_config]
+  exposed_ports = { pprof = "6060", node_rpc = "1234", miner_rpc = "2345" }
```

Comment:
We should really have ports inherited from the manifest. We should be able to just add them once in the manifest...

Reply:
Same applies for the `selector` I'd say.